config/src/main/java/org/springframework/security/config/annotation/method/configuration/EnableReactiveMethodSecurity.java

```diff
@@ -74,4 +74,11 @@
 	 */
 	boolean useAuthorizationManager() default true;
 
+	/**
+	 * Indicate whether to resolve authorization templates with annotation parameters
+	 * @return the resolution policy to use
+	 * @since 6.3
+	 */
+	boolean useAnnotationParameters() default false;
```

Comment:
I don't think that placing the property here makes sense. The property only enables for pre/post but is present on `@EnableReactiveMethodSecurity` which will likely in the future be similar to `@EnableMethodSecurity` which controls JSR 250 & `@Secured` too